packages/google-cloud-apigeeconnect/src/v1/tether_client.ts

```diff
@@ -177,7 +177,8 @@ export class TetherClient {
     this.descriptors.stream = {
       egress: new this._gaxModule.StreamDescriptor(
         this._gaxModule.StreamType.BIDI_STREAMING,
-        !!opts.fallback
+        !!opts.fallback,
+        /* gaxStreamingRetries: */ true
```

Comment:
Can we write in this way? Looks cleaner.

```suggestion
      true /* gaxStreamingRetries: */ 
```

Reply:
Hi @tanvirstreame, this code is autogenerated from the generator (gapic-generator-typescript) and touches all APIs in the library. If you feel strongly, feel free to create an issue here: https://github.com/googleapis/gapic-generator-typescript